pkg/bisect/bisect.go

```diff
@@ -592,26 +592,29 @@ func (env *env) test() (*testResult, error) {
 	bad, good, infra, rep := env.processResults(current, results)
 	res.rep = rep
 	res.verdict = vcs.BisectSkip
-	if infra > len(results)/2 {
-		// More than 1/2 of runs failed with infrastructure error,
+	if infra*3 > len(results) {
```

Comment:
I think maybe we should also look at flaky flag and number of bad results.
E.g. if we have non-flaky reproducer and got 4 infra errors + 6 bad results (all that did not fail due to infra showed the crash), this looks like we can pretty reliably conclude that this is a bad commit.
Or infra failures always persist for a long time? Can this this be a one-off short spike?